src/Data/Vec/Functional/Algebra/Base.agda

```diff
@@ -0,0 +1,139 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some Vector-related module Definitions
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Vec.Functional.Algebra.Base where
+
+open import Level using (Level; suc; _⊔_)
+open import Function using (_$_)
+open import Data.Nat using (ℕ)
+open import Data.Fin using (Fin)
+open import Data.Vec.Functional
+open import Algebra.Core
+open import Algebra.Bundles
+open import Algebra.Module
+open import Relation.Binary using (Rel)
+open import Data.Vec.Functional.Relation.Binary.Pointwise using (Pointwise)
+
+private variable
+  a ℓ : Level
+  A : Set a
+  n : ℕ
+
+------------------------------------------------------------------------
+-- Vector-lifted relations & operators
+
+pointwiseᵛ : ( _≈_ : Rel A ℓ ) → Rel (Vector A n) ℓ
+pointwiseᵛ _≈_ = Pointwise _≈_
+
+zipWithᵛ : Op₂ A → Op₂ (Vector A n)
+zipWithᵛ _∙_ = zipWith _∙_
+
+mapᵛ : Op₁ A → Op₁ (Vector A n)
+mapᵛ f = map f
```

Comment:
Ditto. (and all the more so, now that 3 imported definitions are being subject to the same treatment...)